insuree_batch/templates/insuree_batch/front.svg

```diff
@@ -0,0 +1,2630 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
If this is the template from TZ implementation then it should be replaced with a generic version. The easier is to have the  TZ template without the bg images, only form in EN and QR code.

Reply:
What is the generic version? Also the template is user defined. If they want it with bg image we have to keep it like that.